src/Services/Testing/Testing.API/Application/Queries/Tasks/TaskQueries.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Dapper;
+using Npgsql;
+using Testing.API.DTOs.Tasks;
+
+namespace Testing.API.Application.Queries.Tasks;
+
+public interface ITaskQueries
+{
+    Task<TaskResponse> GetTaskAsync(Guid id);
+
+    Task<IEnumerable<TaskResponse>> GetAllTasksAsync();
+}
+
+public class TaskQueries : ITaskQueries
```

Comment:
Class and its models must be in one folder.
I usually create inner folder Models